api/src/main/java/io/strimzi/api/kafka/model/KafkaBridgeSpec.java

```diff
@@ -35,7 +35,7 @@
 
     private static final long serialVersionUID = 1L;
 
-    private int replicas;
+    private int replicas = 1;
```

Comment:
Why are you not using the constant here to make clear the intent that replicas is initialized with the default

Reply:
Which constants are you referring to here? The `KafkaBridgeCluster.DEFAULT_REPLICAS`? The access modifier is `protected` so I am afraid I can't use it here.

Reply:
I have created a private constant for this now.

Reply:
I think switching default replicas in the KakaBridgeCluster class to public is perfectly fine given that It is an important implementation detail that it should be exposing to other classes. Unless @scholzj thinks differently on this?

Reply:
Let me know and I will change accordingly. Thanks.